app/views/issued_certificates/verify.html.erb

```diff
@@ -3,6 +3,20 @@
 <% content_for(:head) do %>
   <title>Certificate <%= presenter.serial_number %></title>
   <%= stylesheet_pack_tag 'IssuedCertificatesVerifyPack', media: nil %>
+  <%= javascript_tag nonce: true do %>
+    <%= raw segment.setup_segment_head %>
+  <% end %>
+  <% if current_user %>
+    <%= javascript_tag nonce: true do %>
+      analytics.track(
+      'Course Completed',
```

Comment:
I'm not sure this is the valid event - `Course Completed` would be send to Segment each time user will visit certificate page. This could be send from backend code.

Reply:
That's true. This should e 'Certificate Viewed' if anything.
I'll work on this a bit more to trigger the call from ruby code instead.